### PR TITLE
[onert] Introduce Pass for training

### PR DESCRIPTION
**Pull request by ragmani**
This commit introduces Pass for training.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

File: runtime/onert/core/src/compiler/train/pass/Pass.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_COMPILER_TRAIN_PASS_PASS_H__
+#define __ONERT_COMPILER_TRAIN_PASS_PASS_H__
+
+#include "../../pass/IPass.h"
+
+#include <string>
```
Comment (glistening) [inline runtime/onert/core/src/compiler/train/pass/Pass.h:22]:
```suggestion
```
I don't see any dependency to string from Pass.h.

**Review by glistening — APPROVED** [review]
LGTM except for https://github.com/Samsung/ONE/pull/10891/files#r1236237959

File: runtime/onert/core/src/compiler/train/pass/Pass.h (hunk 1 above)
Comment (ragmani) [reply runtime/onert/core/src/compiler/train/pass/Pass.h]: I removed it.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**